src/utilities.ts

```diff
@@ -51,18 +51,9 @@ export function getMinorFullName(acronym: string): string {
 }
 
 export function getGradFullName(acronym: string | undefined): string {
-  // TODO: connect requirements side here instead of using dummy data
-  const acronymDict: Record<string, string> = {
-    TEMP: 'TEMP PROGRAM',
-    TEMP2: 'TEMP PROGRAM 2',
-    TEMP3: 'TEMP PROGRAM 3',
-  };
-
-  return acronym ? acronymDict[acronym] : '';
-
   // Return empty string if grad is not in requirementJSON
-  // const grad = acronym ? requirementJSON.grad[acronym] : null;
-  // return grad ? grad.name : '';
+  const grad = acronym ? requirementJSON.grad[acronym] : null;
+  return grad ? grad.name : '';
```

Comment:
The name of the grad program doesn't seem to show on req bar:
![Screen Shot 2021-05-17 at 9 56 30 PM](https://user-images.githubusercontent.com/54298311/118578761-bd8ce800-b75a-11eb-955b-48c042d5d073.png)

@einc is this intended behavior or do we want to show the name of the grad program?
In the case that we do want to show the name, I think you'll have to modify RequirementHeader.vue, @willespencer.